### PR TITLE
Update copy on rollover page

### PR DESCRIPTION
**Pull request by malcolmbaig**
### Context

The copy has been updated in the Publish prototype.

https://trello.com/c/wh33iXZr

### Changes proposed in this pull request

Match updates made in the Publish prototype.

**Before**
Screenshot_20210623_083318-1

**After**

Screenshot_20210624_121932

### Guidance to review

Page is visible at `/rollover`.

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally
- [ ] Product Review

### CODE REVIEW

**Review by TobyRet — APPROVED** [review]
LGTM

**Review by stevehook — APPROVED** [review]
Tested locally. Looks good.

**Comment by defong** [comment]
image

Is there more to be updated?

File: app/views/pages/rollover.html.erb [hunk 1]
```diff
@@ -7,19 +7,29 @@
...
 
-    <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-6">
-      <li>add any new courses</li>
-      <li>delete courses you're no longer running</li>
-      <li>edit titles, outcomes and locations where necessary</li>
+    <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-4">
+      <li>degree requirements</li>
+      <li>GCSE requirements</li>
+      <li>if you can sponsor visas</li>
+      <li>locations of school placements</li>
```
Comment (paulrobertlloyd) [inline app/views/pages/rollover.html.erb:26]: Not sure if we should be saying this now, given we’re not adding categories, and only removing the 37 limit. Am conspiring with @B-Bonner on what we should say instead.

File: app/views/pages/rollover.html.erb [hunk 2]
```diff
@@ -7,19 +7,29 @@
     </h1>
 
     <p class="govuk-body">
-      All your courses, locations and details have been copied over
-      from the current recruitment cycle (<%= current_recruitment_cycle_period_text %>). You can now
-      update them for <%= next_recruitment_cycle_period_text %>.
+      Courses for the next cycle (<%= next_recruitment_cycle_period_text %>) will be published on 1 September 2021.
```
Comment (frankieroberto) [inline app/views/pages/rollover.html.erb:10]: The "1 September 2021" date was a placeholder for testing. We shouldn't publish this without confirming the actual date first.

**Review by frankieroberto — CHANGES_REQUESTED** [review]

File: app/views/pages/rollover.html.erb (hunk 2 above)
Comment (malcolmbaig) [reply app/views/pages/rollover.html.erb:10]: > Is there more to be updated?

@defong Among the cards in this sprint on the Candidate board, nothing that relates to that partial. Maybe a question for @paulrobertlloyd / @frankieroberto ?

**Comment by malcolmbaig** [comment]
> `September` thinking
> 
> Is there more to be updated?
> 
> image


@defong Among the cards in this sprint on the Candidate board, nothing that relates to that partial. Maybe a question for @paulrobertlloyd / @frankieroberto ?

File: app/views/pages/rollover.html.erb (hunk 2 above)
Comment (paulrobertlloyd) [reply app/views/pages/rollover.html.erb:10]: Can we make this date a constant somewhere? Then here we can format that date as above, and for the ‘Publish in October’ button we can use that same date, but formatted as the month. Make sense? @malcolmbaig @defong

**Comment by paulrobertlloyd** [comment]
> `September` thinking

Am reliably informed that it will still be October when Find opens for the next cycle.

File: app/views/pages/rollover.html.erb (hunk 2 above)
Comment (B-Bonner) [reply app/views/pages/rollover.html.erb:10]: Should we change it to 'in October' then?

Comment (frankieroberto) [reply app/views/pages/rollover.html.erb:10]: @paulrobertlloyd @B-Bonner will we not have a date confirmed by the time rollover starts? Feels like we ought to be able to tell providers this!?

Comment (paulrobertlloyd) [reply app/views/pages/rollover.html.erb:10]: Being confirmed today, I’m told.

File: app/views/pages/rollover.html.erb [hunk 3]
```diff
@@ -7,19 +7,29 @@
...
+    <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-4">
+      <li>degree requirements</li>
+      <li>GCSE requirements</li>
+      <li>if you can sponsor visas</li>
+      <li>locations of school placements</li>
     </ul>
 
+    <p class="govuk-body">
+      You can also add, edit or delete courses.
+    </p>
```
Comment (paulrobertlloyd) [inline app/views/pages/rollover.html.erb:31]: @malcolmbaig Updated content:

```suggestion
    </ul>

    <p class="govuk-body">
      You can also add, edit or delete courses and locations.
    </p>
```

Comment (B-Bonner) [reply app/views/pages/rollover.html.erb:31]: Yep, looks good.

File: app/views/pages/rollover.html.erb (hunk 2 above)
Comment (paulrobertlloyd) [reply app/views/pages/rollover.html.erb:10]: 5 October 2021.

Comment (malcolmbaig) [reply app/views/pages/rollover.html.erb:10]: Will move that into a constant.

**Review by B-Bonner — APPROVED** [review]
Approving those final changes:
1) changing the date to 5 October 2021 
2) changing the copy in the bulled list and the line beneath it

**Comment by B-Bonner** [comment]
The date on this page needs changing too @malcolmbaig (mentioning in case it isn't covered by your changes).

Screenshot 2021-06-24 at 11 09 49

**Comment by malcolmbaig** [comment]
> The September date needs changing on this page too @malcolmbaig (mentioning in case it isn't covered by your changes).


Thanks for mentioning, that page isn't covered by the Trello card. Will push some changes up shortly so it's done as part of this PR.

**Comment by B-Bonner** [comment]
Great, thanks @malcolmbaig

**Comment by malcolmbaig** [comment]
New commits added which:

- Put the next cycle open date in Settings
- Use this setting wherever we reference that date (applying the correct formatting in each case).

@paulrobertlloyd @frankieroberto @B-Bonner @defong

**Review by paulrobertlloyd — APPROVED** [review]
LGTM

**Review by frankieroberto — APPROVED** [review]

**Review by TobyRet — APPROVED** [review]

**Pull request merged**